src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -38,6 +36,27 @@ class SettingsMigration : Activity {
     RunOnceUtil.runOnceForApp("ToggleCodyToolWindowAfterMigration") {
       ApplicationManager.getApplication().invokeLater { toggleCodyToolbarWindow(project) }
     }
+    RunOnceUtil.runOnceForApp("CodyAccountsIdsRefresh") {
```

Comment:
wow, what's that? can you tell me a few words about `runOnceForApp`? 😮

Reply:
Like IJ documentation says `Perform the task if it was not performed before for this application.`  From the code i can tell you that it stores id and boolean value in `BasePropertyService`